hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/TestWriteMergeOnRead.java

```diff
@@ -213,6 +213,14 @@ protected Map<String, String> getMiniBatchExpected() {
     return expected;
   }
 
+  @Override
+  protected void testConcurrentCommit(TestHarness pipeline) throws Exception {
+    pipeline.checkpoint(1)
+        .assertNextEvent()
```

Comment:
I'm confused by the method name, is it a concurrent commit? The pipeline only commits 1 commit.